storage/src/main/java/org/apache/uniffle/storage/handler/impl/HdfsFileReader.java

```diff
@@ -92,4 +94,11 @@ public synchronized void close() throws IOException {
   public Path getPath() {
     return path;
   }
+
+  public long getFileLen() throws IOException {
+    if (fileLen == -1) {
```

Comment:
We could call `getFileLen` multiple times. If the data is flushed after we read the data at the first time. The length should vary.